packages/renderer/src/lib/pod/PodDetailsSummary.svelte

```diff
@@ -18,6 +22,7 @@ onMount(async () => {
       const getKubePod = await window.kubernetesReadNamespacedPod(pod.name, ns);
       if (getKubePod) {
         kubePod = getKubePod;
+        events = $kubernetesCurrentContextEvents.filter(ev => ev.involvedObject.uid === getKubePod.metadata?.uid);
```

Comment:
Is this reactive ? I feel like this is not a derived state, and will only be called on the mount ?

Reply:
changing to svelte5 and making it reactive